docs/reference/analysis/overview.asciidoc

```diff
@@ -0,0 +1,75 @@
+
+== Text analysis overview
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+Text analysis enables {es} to perform full-text search, where the search returns
+all _relevant_ results rather than just exact matches.
+
+For example, a full-text search for `Quick fox jumps` should not only return
+documents containing that exact search string but also documents containing
+similar or related words, like  `fast fox` or even `foxes leap`.
+
+[discrete]
+[[tokenization]]
+=== Tokenization
+
+Analysis makes full-text search possible by breaking an text down into smaller
+chunks, called _tokens_. In most cases, these tokens are individual words.
+
+For example, without analysis, the text `Quick brown fox` can only be
+matched by searches for the exact string `Quick brown fox`. With analysis,
+the text is converted to the tokens `[Quick, brown, fox]`, which can
+be matched by searches for `Quick fox`, `fox brown`, or other variations.
+
+While improved, this example search experience still has a few problems:
```

Comment:
I'd be inclined to push the bulk of this transition to the beginning of the following section.

"This example search experience..." is kind of awkward. I'd be inclined to say something like"

> Tokenization enables matching on individual terms, but each token is still matched literally. This means that a search for "quick" won't match "Quick", and searching for "foxes" won't match "fox". You can improve matching by normalizing the tokens to a standard form.

Reply:
I reworded as suggested and moved most of this content into the `Normalization` section: 8b6766a